src/gui/diffshow.rs

```diff
@@ -84,7 +85,7 @@ impl DiffShow {
         }
 
         if !self.first_lines_of_changed_blocks.is_empty() {
-            self.galley = Some(ui.ctx().fonts(|f| f.layout_job(layout_job)));
+            self.galley = Some(ui.ctx().fonts_mut(|f| f.layout_job(layout_job)));
```

Comment:
Why `fonts_mut`? We don't need mutation, and the function `fonts`[still exists](https://docs.rs/egui/latest/egui/struct.Context.html#method.fonts).